invenio_app_ils/locations/loaders/jsonschemas/location.py

```diff
@@ -8,8 +8,53 @@
 """Location schema for marshmallow loader."""
 
 from invenio_records_rest.schemas import RecordMetadataSchemaJSONV1
-from invenio_records_rest.schemas.fields import PersistentIdentifier
-from marshmallow import fields
+from invenio_records_rest.schemas.fields import (DateString,
+                                                 PersistentIdentifier)
+from marshmallow import (EXCLUDE, Schema, ValidationError, fields, post_load,
+                         pre_load, validates)
+
+_weekday_names = ["monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"]
```

Comment:
Uppercase? Since we are using this in cli.py too, what about putting it in common somewhere? e.g. in the closure module, in `api.py` or `constants.py`?

Reply:
(won't fix)